src/WebJobs.Script/OutOfProc/Http/HttpScriptInvocationContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.WebJobs.Script.OutOfProc.Http
+{
+    public class HttpScriptInvocationContext
+    {
+        public IDictionary<string, object> Data { get; set; } = new Dictionary<string, object>();
+
+        public IDictionary<string, object> Metadata { get; set; } = new Dictionary<string, object>();
```

Comment:
This is BindingData, right? Perhaps we should name it as such. That way if we want to pass other general metadata in the future on this context, we won't have a naming issue?

Reply:
resolved over offline discussion. It makes sense to keep this generic Data/Metadata instead of input/output bindings